webapp/channels/package.json

```diff
@@ -105,6 +105,7 @@
     "@hot-loader/react-dom": "17.0.2",
     "@mattermost/calls-common": "0.27.0",
     "@mattermost/eslint-plugin": "*",
+    "@mattermost/mmjstool": "github:mattermost/mattermost-utilities#abf31bebf7d678321b8b19d875beeebcc3c126f1",
```

Comment:
I also renamed this package in the package.json to use the namespaced version of the name because, similar to things like `mattermost-plugin-eslint`, someone squatted on the `mmjstool` NPM package and it got locked down by NPM